teamspeak3/teamspeak3.json

```diff
@@ -0,0 +1,75 @@
+{
+  "pufferd": {
+    "type": "Voice",
+    "display": "Teamspeak 3",
+    "install": {
+      "commands": [
+        {
+          "files": "http://dl.4players.de/ts/releases/3.1.1/teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
+          "type": "download"
+        },
+        {
+          "commands": [
+            "tar -xvf teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
+	    "touch teamspeak3-server_linux_amd64/.ts3server_license_accepted"
```

Comment:
In general, when it comes to anything license related, we always require it to be set explicitly, as we do not want to get into issues where we auto-accept licenses for folk who don't know it's there.